allofplos/plos_corpus.py

```diff
@@ -229,19 +229,17 @@ def repo_download(dois, tempdir, ignore_existing=True, plos_network=False):
         existing_articles = [filename_to_doi(file) for file in listdir_nohidden(tempdir)]
         dois = set(dois) - set(existing_articles)
 
-    if dois:
-        for doi in tqdm(sorted(dois)):
-            url = URL_TMP.format(doi)
-            articleXML = et.parse(url)
-            article_path = doi_to_path(doi, directory=tempdir)
-            # create new local XML files
-            if ignore_existing is False or ignore_existing and os.path.isfile(article_path) is False:
-                with open(article_path, 'w') as file:
-                    file.write(et.tostring(articleXML, method='xml', encoding='unicode'))
-                if not plos_network:
-                    time.sleep(1)
-    else:
-        pass
+    for doi in tqdm(sorted(dois), disable=len(dois)==0):
```

Comment:
you can change disable=len(dois)==0 to disable=not dois
because this parameter needs a bool argument, and if dois is not 0, is equivalent to True, with the not, is false, so it will work as long a the list is not empty